src/Raven.Server/Documents/Indexes/Index.cs

```diff
@@ -665,6 +665,16 @@ private StorageEnvironmentOptions CreateStorageEnvironmentOptions(DocumentDataba
 
             var indexTempPath = configuration.TempPath?.Combine(name);
 
+            if (configuration.RunInMemory == false)
+            {
+                string disableMarkerPath = indexPath.Combine("disable.marker").FullPath;
+                if (File.Exists(disableMarkerPath))
+                {
+                    throw new IndexOpenException("Unable to open index: " + name + ", it has been manually disabled via the file: '" + disableMarkerPath +"'." +
```

Comment:
better to use interpolation and wrap the name in '' maybe?

Reply:
Done